OracleDatabase/RAC/OracleConnectionManager/dockerfiles/21.3.0/client21c_install.rsp

```diff
@@ -0,0 +1,104 @@
+###############################################################################
+## Copyright(c) Oracle Corporation 2022.                                     ##
```

Comment:
This should be: `Copyright(c) 2022 Oracle and/or its affiliates.`

Reply:
Added the fix in a new commit  "Added fix suggested for the pull request #2336".